amazon-dynamodb-quickstart/README.md

```diff
@@ -0,0 +1,127 @@
+
+# DynamoDB local instance
+
+Just run it as follows:
+`docker run --rm --name local-dynamo -p 8000:4569 -e SERVICES=dynamodb -e START_WEB=0 -d localstack/localstack`
+
+DynamoDB listens on `localhost:8000` for REST endpoints.
+
+Create an AWS profile for your local instance using AWS CLI:
+
+```
+$ aws configure --profile localstack
+AWS Access Key ID [None]: test-key
+AWS Secret Access Key [None]: test-secret
+Default region name [None]: us-east-1
+Default output format [None]:
+```
+
+## Create table
+
+Create a DynamoDB table using AWS CLI and the localstack profile.
+```
+aws dynamodb create-table --table-name QuarkusFruits \
+                          --attribute-definitions AttributeName=fruitName,AttributeType=S \
+                          --key-schema AttributeName=fruitName,KeyType=HASH \
+                          --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1 \
+                          --profile localstack --endpoint-url=http://localhost:8000
+```
+
+# Run the demo on dev mode
+
+- Run `./mvnw clean package` and then `java -jar ./target/amazon-dynamodb-quickstart-1.0-SNAPSHOT-runner.jar`
+- In dev mode `./mvnw clean quarkus:dev`
+
+Go to [`http://localhost:8080/fruits.html`](http://localhost:8080/fruits.html), it should show a simple App to manage a list of Fruits. 
+You can add fruits to the list via the form.
+
+Alternatively, go to [`http://localhost:8080/async-fruits.html`](http://localhost:8080/async-fruits.html) with the simple App communicating with Async resources.
+
+# Running in native
+
+You can compile the application into a native executable using:
+
+`./mvnw clean install -Pnative`
+
+and run with:
+
+`./target/amazon-dynamodb-quickstart-1.0-SNAPSHOT-runner` 
+
+
+# Running native in container
+
+Build a native image in container by running:
+
+`./mvnw package -Pnative -Dnative-image.docker-build=true`
+
+Build a docker image:
+`docker build -f src/main/docker/Dockerfile.native -t quarkus/amazon-dynamodb-quickstart .`
+
+Create a network that connect your container with localstack
+`docker network create localstack`
+
+Stop your localstack container you started at the begining
+`docker stop local-dynamo`
+
+Start localstack and connect to the network
+`docker run --rm --network=localstack --name localstack -p 8008:4569 -e SERVICES=dynamodb -e START_WEB=0 -d localstack/localstack`
+
+Create Dynamo table
+```
+aws dynamodb create-table --table-name QuarkusFruits \
+                          --attribute-definitions AttributeName=fruitName,AttributeType=S \
+                          --key-schema AttributeName=fruitName,KeyType=HASH \
+                          --provisioned-throughput ReadCapacityUnits=1,WriteCapacityUnits=1 \
+                          --profile localstack --endpoint-url=http://localhost:8000
```

Comment:
Local endpoint has changed to port 8008, please update.

Reply:
actually it's fine, it's wrong port used to start localstack. will fix that